peachjam/models/save_document.py

```diff
@@ -102,6 +102,10 @@ class SavedDocument(models.Model):
 
     objects = SavedDocumentManager()
 
+    def __str__(self):
+        latest_expression = self.work.documents.latest_expression().first()
```

Comment:
It's unusual to put complex logic into a `__str__` because it can get called all over the place. If this is necessary for rendering, use a specific method.